action.yml

```diff
@@ -53,11 +53,11 @@ inputs:
     description: nodejs version to use
     required: false
     default: "20.x"
-  installAdmin:
+  install-admin:
```

Comment:
@shyim @philipreinken this might clash with https://github.com/shopware/github-actions/blob/8a2de1a1d508cf7035e9fbc5a0013f012bfd3d7f/setup-extension/action.yml#L98 no ?

Reply:
Yes, it does. We'll need to change these inputs in all downstream actions 👍 Thanks for the reminder!

Reply:
done already in github-actions repository

Reply:
Ah, it's [happened already](https://github.com/shopware/github-actions/commit/92663f9baa618d6a95d2fbee079b3f6e35b22f5b), thanks @shyim 😄